python/mxnet/rnn/rnn_cell.py

```diff
@@ -355,14 +355,20 @@ class LSTMCell(BaseRNNCell):
     params : RNNParams or None
         container for weight sharing between cells.
         created if None.
+    forget_bias : bias added to forget gate, default 0.0.
+        Jozefowicz et al. 2015 recommends setting this to 1.0
+        but the default is set to 0.0 for backward compatibility.
+
     """
-    def __init__(self, num_hidden, prefix='lstm_', params=None):
+    def __init__(self, num_hidden, prefix='lstm_', params=None, forget_bias=1.0):
```

Comment:
OK. I brought this up mostly from engineering perspective, as it's generally not recommended to change the default behavior, since codes that are already using this API have no way of knowing the changes underneath. New RNNParams class seems to be a natural way of encapsulating new changes while maintaining the current behavior.

That said, I'm fine either way since users would most likely welcome a better default parameter. API-wise, keep in mind that once you add a new argument, there's really no turning back, because removing the argument would break codes that started to use it. Semantic-wise, though I prefer keeping parameter initialization in the parameter class, OOP-wise I think it's still OK to say that "parameter initialization is the concern of an LSTM cell".